### DISCUSSION TITLE
Add Chains: Fantom and Fantom Testnet

### DISCUSSION THREAD

**Discussion by taycaldwell** [wevm/wagmi#916] [Ideas]
Feature request to add chain(s):

- Fantom
- Fantom Testnet

I can open a PR to add it once I get the +1.

**Comment by 0xkaz** [comment]
I would like to contribute it, but alchemy ( and etc ) are not supporting fantom yet. 

If you guys accept me to use public node ( e.g. https://rpc.ftm.tools/ ) instead of node services, even though specify alchemy, I will be able to support it soon.

CC @taycaldwell

**Reply by GrizzlyWebDev** [reply]
Hey @0xkaz I used this method to add bsc and it may work for phantom as well! hope it helps! 

https://github.com/wagmi-dev/wagmi/discussions/1051

**Reply by 0xkaz** [reply]
@GrizzlyWebDev 

Thanks for the comment. 
And sorry to be late. I forgot to follow this thread. 😓 

Okay, I will try to handle it when I have time.

**Comment by jxom** [comment]
With the introduction of `0.9.x`, wagmi now comes with the `fantom` & `fantomTestnet` chains built-in (re-exported from our [`@wagmi/chains` reference repository](https://github.com/wagmi-dev/references/tree/main/packages/chains)). Read more [here](https://wagmi.sh/react/chains).
